get-dcos-docker.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Launches a DC/OS dev environment based containing a running 3-agent cluster and dev tools.
+#
+# Prerequisites:
+# - 8GB RAM
+# - Vagrant
+# - Virtualbox
+#
+# Arguments:
+# - BOX_PATH: Local path to use for .box image (useful if installing from USB stick)
+
+# capture anonymous metrics for reporting
+curl https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/sdk/create-dev-env-start.png >/dev/null 2>&1
```

Comment:
Open question: This is reusing the location used by build-dcos-image.sh, should it be different?